### ISSUE TITLE
Does not log in the client's console

### ISSUE THREAD

**Issue by maximzavadskiy** [farpoint/meteor-winston-client#4]
Using it like `Winston.log( 'info', 'Winston initialized')` does not output anything to the client's console

**Comment by brentjanderson** [comment]
It should appear on the Server, the transport is intended to push log information to a centralized collector for logging in a Winston transport.

**Comment by maximzavadskiy** [comment]
Right, probably I interpreted this sentence wrong 'logs messages to the console and sends them up to the server as well'. But then which console is mentioned here?

**Comment by brentjanderson** [comment]
That is an error. I’ve moved off of using Winston Client in my own projects, but leave this ticket open and I can make some adjustments to properly log on the console in addition to forwarding to the server for collection.

**Comment by maximzavadskiy** [comment]
Brent, could you tell, what is the reason you are not using Winston for ur meteor projects? I was looking myself for a framework that allows good logging possibilities with a feature of logging from the client to the server. But there is a lot of options out there, and no official one.

**Comment by brentjanderson** [comment]
To be clear, I use Winston on the server, I just don’t use winston-client, which is a transport for Winston, enabling you to push browser logs to the server.

**Comment by noahsw** [comment]
Any updates on this ticket? For debugging, it's helpful to see logs in the console.
